src/apps/tenderize/ethereum/tenderize.tender-tokens.token-fetcher.ts

```diff
@@ -0,0 +1,85 @@
+import { Inject } from '@nestjs/common';
+import Axios from 'axios';
+
+import { IAppToolkit, APP_TOOLKIT } from '~app-toolkit/app-toolkit.interface';
+import { Register } from '~app-toolkit/decorators';
+import { buildDollarDisplayItem } from '~app-toolkit/helpers/presentation/display-item.present';
+import { ContractType } from '~position/contract.interface';
+import { PositionFetcher } from '~position/position-fetcher.interface';
+import { AppTokenPosition } from '~position/position.interface';
+import { Network } from '~types/network.interface';
+
+import { TenderizeContractFactory } from '../contracts';
+import { ethereumEndpoint } from '../helpers/constants';
+import { tenderTokenFetcherQuery, TenderTokenFetcherResponse } from '../helpers/queries';
+import { APYResponse } from '../helpers/types';
+import { TENDERIZE_DEFINITION } from '../tenderize.definition';
+
+const appId = TENDERIZE_DEFINITION.id;
+const groupId = TENDERIZE_DEFINITION.groups.tendertokens.id;
```

Comment:
Rename group to `tender`, and rename the file to `tenderize.tender.token-fetcher.ts`

Rename the class below to `EthereumTenderizeTenderTokenFetcher`